lib/mongo_dart.dart

```diff
@@ -4,48 +4,77 @@
 
 library mongo_dart;
 
-import 'dart:io';
 import 'dart:async';
+import 'dart:collection';
+import 'dart:io' hide FileMode;
 import 'dart:math';
+import 'dart:typed_data';
+import 'package:collection/collection.dart';
 import 'package:crypto/crypto.dart' as crypto;
-import 'dart:collection';
+import 'package:dart2_constant/convert.dart';
+import 'package:dart2_constant/io.dart';
 import 'package:bson/bson.dart';
 import 'package:logging/logging.dart';
 import 'package:mongo_dart_query/mongo_dart_query.dart';
 import 'package:pool/pool.dart';
-import 'dart:convert';
-import 'dart:typed_data';
-import "package:collection/collection.dart";
 
 export 'package:bson/bson.dart';
 export 'package:mongo_dart_query/mongo_dart_query.dart';
 
-part 'src/network/packet_converter.dart';
-part 'src/network/connection_manager.dart';
-part 'src/network/connection.dart';
-part 'src/network/mongo_message_transformer.dart';
+part 'src/connection_pool.dart';
```

Comment:
lol what is this new formatting?

Reply:
I sorted the imports and parts into alphabetical order, IIRC it's part of the Dart style guide.

On second though, there were definitely a few lines I forgot to sort, so it looks weird for sure, haha.